server/models/plant.ts

```diff
@@ -0,0 +1,80 @@
+import {mongoose} from './example'; 
+//import {Species} from './species';
+//import {Comment} from './comment';
+//import {Toxicity} from './toxicity';
```

Comment:
I don't think we should import comment nor toxicity
It's the same table in our erd diagram, I would include those in the same file